eks-cluster.yaml

```diff
@@ -32,10 +26,6 @@ iam:
       wellKnownPolicies:
         autoScaler: true
 
-availabilityZones:
```

Comment:
How come these were removed? I ask because I am currently getting:

```
2022-06-10 10:36:22 [✖]  AWS::EKS::Cluster/ControlPlane: CREATE_FAILED – "Resource handler returned message: \"Cannot create cluster 'gitpod-pr-53' because us-east-1e, the targeted availability zone, does not currently have sufficient capacity to support the cluster. Retry and choose from these availability zones: us-east-1a, us-east-1b, us-east-1c, us-east-1d, us-east-1f (Service: Eks, Status Code: 400, Request ID: 0a6d1672-b06e-4fdc-8591-e2018d2c8ed2)\" (HandlerErrorCode: InvalidRequest)"
```

Reply:
In trying to remove hard coded components that users need to change based on where they run the guide - it was more likely someone would change 'us-west-2' to eu-west-1 and not change the availability zones further down the file, we removed specifying the AVZ directly and it defaults to using all of them - most ones are 1-3, only us-east-1 is that sprawling (also why us-west-2 was defaulted and would be a better starting point - us-east-1 is the default in so many configurations it also always the most resource constrained region)